docs/source/validate.rst

```diff
@@ -34,26 +34,19 @@ validate a NeXus file
 
 .. code-block:: console
    :linenos:
-   
-   usage: punx validate [-h] [--report REPORT] [-l [LOGFILE]] [-i INTEREST]
-                        infile
-   
+
+   usage: punx validate [-h] [--report REPORT] infile
+
    positional arguments:
-     infile                HDF5 or NXDL file name
-   
+     infile           HDF5 or NXDL file name
+
    optional arguments:
-     -h, --help            show this help message and exit
-     --report REPORT       select which validation findings to report, choices:
-                           COMMENT,ERROR,NOTE,OK,TODO,UNUSED,WARN
-     -l [LOGFILE], --logfile [LOGFILE]
-                           log output to file (default: no log file)
-     -i INTEREST, --interest INTEREST
-                           logging interest level (1 - 50), default=1 (Level 1)
+     -h, --help       show this help message and exit
+    --report REPORT  select which validation findings to report, choices: COMMENT,ERROR,NOTE,OK,OPTIONAL,TODO,UNUSED,WARN (separate with comma if more
+                     than one, do not use white space)
 
 The **REPORT** findings are as presented in the table above for each validation step.
 
```

Comment:
Seems kinda annoying that you have to maintain the help text in two places. No suggestion; just an observation.

Reply:
agreed